### DISCUSSION TITLE
Repository Activity filter is a spam of "issue labelled" events

### DISCUSSION THREAD

**Discussion by douglascamata** [community/community#66333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I starred a bunch of projects that use bots to automatically label issues and PRs that are created in the project. 

The "Repository Activity" filter in the Feed then generates a huge spam nightmare of events for each issue being automatically labelled by bots.

**Comment by r-thomson** [comment]
Stars have long been a way to "bookmark" a repo on GitHub. Now that stars are linked to activity in the feed, are we supposed to stop using stars that way?

Why would I care about issue activity in a repo that I'm not watching or contributing to? It's unnecessary spam.
